### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by irctrakz**
Remove setuid for ping supporting older hosts (NAS) that don't have extended attributes.

## :recycle: Current situation

root@HomeNAS ~ # docker pull homebridge/homebridge:latest
latest: Pulling from homebridge/homebridge
0622fac788ed: Pull complete
70913027ecf4: Extracting [==================================================>]  223.5MB/223.5MB
06c8ff15a4f2: Download complete
726e97df8c02: Download complete
5555af05ef06: Download complete
63cccba96eec: Download complete
a99a466f3aad: Download complete
failed to register layer: lsetxattr security.capability /usr/bin/ping: operation not supported

## :bulb: Proposed solution

Remove setuid, it's not required when the container has host networking.

## :gear: Release Notes

This is a bugfix and should have no impact

## :heavy_plus_sign: Additional Information

_If applicable, provide additional context in this section._

### Testing

_Which tests were added? Which existing tests were adapted/changed? Which situations are covered, and what edge cases are missing?_

### Reviewer Nudging

https://github.com/homebridge/docker-homebridge/blob/2025-05-16/Dockerfile#L37

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ RUN set -x \
   && apt-get install -y python3 python3-pip pipx python3-setuptools git make g++ libnss-mdns \
     avahi-discover libavahi-compat-libdnssd-dev python3-venv python3-dev \
   && pipx install tzupdate \
```
[the remark this answers is not in the document]
Comment (irctrakz) [reply Dockerfile:36]: This is already set in the documentation (compose config): https://github.com/homebridge/docker-homebridge/wiki/Homebridge-on-Portainer

[the remark this answers is not in the document]
Comment (irctrakz) [reply Dockerfile:36]: network_mode: host

**Review by NorthernMan54 — APPROVED** [review]
